src/templates/partials/fold-jeep.hbs

```diff
@@ -1,22 +1,29 @@
 <li id="jeep">
   <div>
     <div class="parallaxBackground"></div>
-
     <div class="aside">
       <img class="line-down" src="/assets/img/icons/line-down_animated.svg">
-      <h3>Jeep anniversary <br/>wrap promo</h3>
+      <h3>Jeep anniversary<br/>wrap promo</h3>
       <img class="line-dot" src="/assets/img/icons/line-down-dot-short-animated.svg">
-
       <ul>
         <li>Illustration</li>
         <li>Vehicle Signage</li>
       </ul>
       <img class="flipper" src="/assets/img/icons/small-arrow-down_animated.svg">
-
       <a href="/feature-project/jeep/">See full project</a>
     </div>
     <div class="images">
-      <img id="jeepImage" src="/assets/img/landing/jeep/jeep@2x.png" alt="">
+      {{!-- The <picture> element does not exist on IE. Instead it will fallback to the <img> --}}
+      {{!-- This image is 62vw wide. So in order to use the smallest possible image, we show a 1000px wide image only at 1000px/0.62vw = 1612px. --}}
+      <picture id="jeep-image">
+        <source srcset="/assets/img/landing/jeep/jeep-400w.png" media="(max-width: 645px)">
+        <source srcset="/assets/img/landing/jeep/jeep-600w.png" media="(max-width: 967px)">
+        <source srcset="/assets/img/landing/jeep/jeep-800w.png" media="(max-width: 1290px)">
+        <source srcset="/assets/img/landing/jeep/jeep-1000w.png" media="(max-width: 1612px)">
+        <source srcset="/assets/img/landing/jeep/jeep-1200w.png" media="(max-width: 1935px)">
+        <source srcset="/assets/img/landing/jeep/jeep-1400w.png" media="(max-width: 2258px)">
+        <img src="/assets/img/landing/jeep/jeep-1600w.png" alt="Custom hand drawn vehicle signage wrapper on a Jeep by Hannah Sutton">
+      </picture>
```

Comment:
So the `<picture>` element wraps a `<img>` and a set of `<source>`. If the condition of the first source is met (in this case `max-width:645px`) then its `srcset` will be used in the `<img>`. In case false it will try the next source and cascade all the way down to fall back on the original `<img>`.

This is allows us to make `<img>` responsive just like we do in CSS.

Browsers that do not support `<picture>` or `srcset` will gracefully ignore all and fall back to the `<img>`. That means IE will always load a gigantic image. Their bad.

@richardverheyen

Reply:
Please also always fill out the `alt` with keywords that describe the images and match the sites content. It's not exact science, but anything is better than blank. Allows our images to pop in Google image searches.